packages/deployer/subtasks/deploy-modules.js

```diff
@@ -8,16 +9,16 @@ const { SUBTASK_DEPLOY_CONTRACTS, SUBTASK_DEPLOY_MODULES } = require('../task-na
 subtask(SUBTASK_DEPLOY_MODULES).setAction(async (_, hre) => {
   logger.subtitle('Deploying modules');
 
-  const modules = filterObject(hre.deployer.data.contracts, (c) => c.isModule);
+  const modules = filterObject(hre.deployer.deployment.contracts, (c) => c.isModule);
 
   const deployedSomething = await hre.run(SUBTASK_DEPLOY_CONTRACTS, {
     contracts: modules,
   });
 
   if (!deployedSomething) {
     logger.info('No modules need to be deployed, clearing generated files and exiting...');
-    logger.info(`Deleting ${hre.deployer.file}`);
-    rimraf.sync(hre.deployer.file);
+    logger.info(`Deleting ${hre.deployer.deployment.file}`);
+    rimraf.sync(path.resolve(hre.config.paths.root, hre.deployer.deployment.file));
```

Comment:
`hre.deployer.deployment.file` is relative, so that it doesn't cause git changes from local paths.
So we have to un-relativize it here.